src/EchoToImageLUT.hpp

```diff
@@ -13,6 +13,9 @@ namespace radar_base {
         std::vector<int> m_data_index;
 
         int m_sweep_size;
+        int m_num_angles;
+        float m_beam_width;
+        int m_window_size;
```

Comment:
I would declare all those and `m_sweep_size` as const, to make it explicit that there is no way to change a LUT during its life cycle. And maybe made them public, so there is no need to create getters in the future